extruct/tool.py

```diff
@@ -56,10 +67,23 @@ def main():
         default=False,
         help='Extract RDFa metadata from the page.',
     )
+    parser.add_argument(
+        '--microformat',
+        action='store_true',
+        default=False,
+        help='Extract microformat metadata from the page.',
+    )
+    parser.add_argument(
+        '--opengraph',
+        action='store_true',
+        default=False,
+        help='Extract opengraph metadata from the page.',
+    )
     args = parser.parse_args()
 
-    if any((args.microdata, args.jsonld, args.rdfa)):
-        metadata = metadata_from_url(args.url, args.microdata, args.jsonld, args.rdfa)
+    if any((args.microdata, args.jsonld, args.rdfa, args.microformat, args.opengraph)):
+        metadata = metadata_from_url(args.url, args.microdata, args.jsonld,
```

Comment:
Could you please use keyword arguments here? It is very easy to make a mistake otherwise.